pkg/state/state.go

```diff
@@ -488,7 +488,20 @@ func (st *HelmState) reformat(spec *ReleaseSpec) []string {
 	var needs []string
 	releaseInstalledInfo := make(map[string]bool)
 	for _, r := range st.OrginReleases {
-		releaseInstalledInfo[r.Name] = r.Desired()
+		var kubecontext, namespace string
+
+		if st.OverrideKubeContext != "" {
+			kubecontext = st.OverrideKubeContext
+		} else {
+			kubecontext = r.KubeContext
+		}
+		if st.OverrideNamespace != "" {
+			namespace = st.OverrideNamespace
+		} else {
+			namespace = r.Namespace
+		}
+
```

Comment:
maybe we can make you code more simple.

Reply:
I'm not sure how to make this simpler, so please do let me know if you have ideas.

Reply:
```suggestion
        kubecontext = r.KubeContext
		if st.OverrideKubeContext != "" {
			kubecontext = st.OverrideKubeContext
		}
		namespace = r.Namespace
		if st.OverrideNamespace != "" {
			namespace = st.OverrideNamespace
		}
```

Reply:
@aarnq